### PR TITLE
Update yanagiba

### PR DESCRIPTION
**Pull request by Adamtaranto**
* [ x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/yanagiba/build.sh [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 
-$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/yanagiba/build.sh]: why have you removed this, I think this is needed: https://bioconda.github.io/linting.html#setup-py-install-args
also for the other recipe

Comment (Adamtaranto) [reply recipes/yanagiba/build.sh]: New recipe was generated with `conda skeleton pypi yanagiba` the build script reverted to the form without flags. 

Have added back in the latest commit.

**Comment by Adamtaranto** [comment]
Conflict with master arose from an edit in another one of my recipes (tirmite) that was accidentally saved into this branch. Fixed by changing timite commits in this branch to match Master (no net change).

**Comment by druvus** [comment]
@Adamtaranto An updated version was added in #6647 so I am closing.

**Pull request closed without merging**